cmd/restic/main.go

```diff
@@ -160,6 +161,8 @@ func main() {
 		exitCode = 10
 	case restic.IsAlreadyLocked(err):
 		exitCode = 11
+	case errors.Is(err, repository.ErrNoKeyFound):
+		exitCode = 12
```

Comment:
The exit codes here are a little jumpy. I'm guessing that the vibe is to use 10+ for random specific error codes. Leaving <10 and >100 for special cases.

Reply:
The 10+ range (probably up to 20) is currently intended for errors related to the repository itself. The 1+ range are general / command errors. There's no really strict approach how to allocate the exitCodes though.

Please update the documentation in `doc/075_scripting.rst` and the help text returned by each command (the easiest way is probably to use search&replace to add exit code 12 to every command that already returns exit code 11).

Reply:
Looks like there's only one block that explains the exit codes, so I added 12 to it. I'm assuming the next release is 0.17.1